1.fee_bumping/fee_bumping.md

```diff
@@ -177,21 +179,132 @@ TODO
 
 TODO
 
-# Introduction to CPFP
-
-TODO
-
-## Overview of how CPFP works
-
-TODO
-
-## History of CPFP (https://github.com/bitcoin/bitcoin/pull/7600)
-
-TODO
-
-## Example of a company using CPFP
-
-TODO
+# Child-Pays-For-Parent
+
+Child Pays for Parent (CPFP) is a wallet feature where a user spends the output
+of an unconfirmed (_parent_) transaction as an input to a new (_child_)
+transaction. The wallet attaches enough fee to the child transaction to
+increase the combined feerate across the parent and child transactions.
+
+## How does CPFP work?
+
+When constructing a new block, miners are incentivized to fill the 1vMB with
```

Comment:
Added as a footnote for now (and since github flavored markdown doesn't support footnotes, it's just a note saying "look at footnote). I think as the scaling book gets expanded, we're going to use vB all over the place, so I've also left a comment to move the footnote to a more appropriate place in the book later.